ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -256,6 +256,34 @@ $defs:
             keys:
               profile:
                 type: str
+          uplink_port_channel_id:
+            documentation_options:
+              table: node-type-uplink-configuration
+            type: int
+            convert_types:
+              - str
+            min: 1
+            max: 999999
+            description: |
+              Only applicable when uplink_type == "port-channel".
+              By default the uplink Port-channel ID will be set to the number of the lowest member interface defined in the uplink_interfaces.
+              This option overrides the default behavior and statically sets the local Port-channel ID.
+              Note! Make sure that an ID is used which does not overlap with the autogenerated Port-channel IDs in the Network Services.
+              Note! For MLAG pairs IDs between 1 and 2000 are supported.
+          uplink_switch_port_channel_id:
+            documentation_options:
+              table: node-type-uplink-configuration
+            type: int
+            convert_types:
+              - str
+            min: 1
+            max: 999999
```

Comment:
I think these are platform dependent on top of mlag dependent. So maybe leave them out.